docs/developer-guide/api.rst

```diff
@@ -326,3 +326,67 @@ If a CREATE, DROP, or TERMINATE statement returns a command status with state QU
         "status": "SUCCESS",
         "message":"Stream created and running"
       }
+
+
+Terminate A Cluster
+-------------------
+
+If you don't need your KSQL cluster anymore, you can terminate the cluster and clean up the resources using this endpoint. When a server receives a ``TERMINATE CLUSTER`` request at ``/ksql/terminate`` endpoint, it writes a ``TERMINATE CLUSTER`` command into the command topic.
+Note that ``TERMINATE CLUSTER`` request can only be sent via the ``/ksql/terminate`` endpoint and you cannot send it via the CLI.
+When each server in the cluster reads a ``TERMINATE CLUSTER`` command, it takes the following steps:
```

Comment:
Document that the cluster should be reduced to a single node before calling terminate?

This also means changing the 'each server in cluster' bit.